app/controllers/validators_controller.rb

```diff
@@ -44,11 +44,6 @@ def home_new
     @network = validators_params[:network]
   end
 
-  # GET /validators-map
```

Comment:
We can also delete this method from routes

Reply:
I thought I did that. Thanks!